example/spec/parallel/testAPI/snsExecutionSpec.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const { s3ObjectExists, deleteS3Object } = require('@cumulus/aws-client/S3');
+const { deleteExecution } = require('@cumulus/api-client/executions');
+const { waitForCompletedExecution } = require('@cumulus/integration-tests');
+const { loadConfig } = require('../../helpers/testUtils');
+const { buildAndStartWorkflow } = require('../../helpers/workflowUtils');
+
+describe('An SNS message is published to the report executions topic', () => {
+  let beforeAllError;
+  let config;
+  let executionKey;
+  let workflowExecutionArn;
+
+  beforeAll(async () => {
+    try {
+      config = await loadConfig();
+
+      process.env.ExecutionsTable = `${config.stackName}-ExecutionsTable`;
+      const workflowExecutionName = 'IngestAndPublishGranule';
+
+      workflowExecutionArn = await buildAndStartWorkflow(
+        config.stackName,
+        config.bucket,
+        workflowExecutionName
+      );
+    } catch (error) {
+      beforeAllError = error;
+    }
+  });
+
+  beforeEach(() => {
+    if (beforeAllError) fail(beforeAllError);
+  });
+
+  afterAll(async () => {
+    await deleteS3Object(config.bucket, executionKey);
+  });
+
+  it('for a deleted execution', async () => {
+    await waitForCompletedExecution(workflowExecutionArn);
+    await deleteExecution({ prefix: config.stackName, executionArn: workflowExecutionArn });
+
+    const executionName = workflowExecutionArn.split(':').pop();
+    executionKey = `${config.stackName}/test-output/${executionName}.output`;
```

Comment:
Upon further investigation, it may be the case that we never have been sending information about deleted executions over to cloud metrics in the first place. 

Consider our publish-executions Lambda:

https://github.com/nasa/cumulus/blob/master/packages/api/lambdas/publish-executions.js#L17

It is just unmarshalling the `NewImage` and sending that in the SNS topic. `NewImage` would only exist for new or updated records.

https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_streams_StreamRecord.html

I guess that all makes sense given that we never previously had a way to delete executions until we added that endpoint. But we would need to discuss with cloud metrics how/if to add support for this before we could do anything, since it would require a breaking change on them. We might want to make sure we have a ticket or architecture topic for it though